Description: Patch for 3.4.0 adapated for 3.3.3 to ensure that header
 files are installed to /usr/include/zookeeper instead of
 /usr/include/c-client-src
Origin: https://issues.apache.org/jira/browse/ZOOKEEPER-1033

Index: zookeeper/src/c/configure.ac
===================================================================
--- zookeeper.orig/src/c/configure.ac	2011-07-13 09:35:58.176129472 +0100
+++ zookeeper/src/c/configure.ac	2011-07-13 09:36:13.716129460 +0100
@@ -3,7 +3,7 @@
 
 AC_PREREQ(2.59)
 
-AC_INIT([zookeeper C client],3.3.3,[zookeeper-user@hadoop.apache.org],[c-client-src])
+AC_INIT([zookeeper C client],3.3.3,[zookeeper-user@hadoop.apache.org],[zookeeper])
 AC_CONFIG_SRCDIR([src/zookeeper.c])
 
 # initialize Doxygen support
